2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1186,75 +1186,96 @@ _Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.3.8_
-        - modifiche effettuate su indicazioni del Verificatore.
+      + *risoluzione dei problemi e attività di miglioramento di processo e prodotto*:
+        - le modifiche devono essere effettuate su indicazioni del Verificatore.
 
-      + stabilire l'indipendenza del controllo della qualità dagli altri processi del ciclo di vita:
-         - il Verificatore è una persona fisica diversa da quella che ha redatto il documento.
+  - *stabilire l'indipendenza del controllo della qualità dagli altri processi del ciclo di vita*:
+         - il Verificatore deve essere una persona fisica diversa da quella che ha redatto il documento.
 
-+ eseguire valutazioni di processi, prodotti e servizi:
++ *eseguire valutazioni di processi, prodotti e servizi*:
 
-  + il Verificatore valuta i prodotti e i servizi al fine di garantire la conformità rispetto ai criteri stabiliti;
-  + il Verificatore si assicura che la verifica e la convalida degli output dei processi del ciclo di vita siano eseguiti conformemente con quanto concordato in precedenza;
-  + il Verificatore applica il processo di misurazione della qualità per verificare che il prodotto o servizio rispetti le metriche precedentemente stabilite;
-  + il Verificatore esprime un giudizio e segnala eventuali problematiche riscontrate.
+  + il Verificatore deve valutare i prodotti e i servizi al fine di garantirne la conformità rispetto ai criteri stabiliti;
+  + il Verificatore deve assicurarsi che la verifica e la convalida degli output dei processi del ciclo di vita siano eseguiti conformemente con quanto concordato in precedenza;
+  + il Verificatore deve applicare il processo di misurazione della qualità per verificare che il prodotto o servizio rispetti le metriche precedentemente stabilite;
+  + il Verificatore deve esprimere un giudizio e segnalare eventuali problematiche riscontrate.
 
-+ gestire rapporti del controllo della qualità:
++ *gestire report e record del Controllo della Qualità*:
 
- - essi consistono in:
+ - *l'attività consiste in*:
 
-  + stilare rapporti relativi alle attività di controllo della qualità:
-    - in sede di retrospettiva il Verificatore aggiorna il gruppo riguardo le attività di controllo qualità dello sprint in corso;
-    - redazione di un verbale contenente anche gli esiti del processo di controllo della qualità.
+  + *stilare report e record relativi alle attività di Controllo della Qualità*:
+    - i report e i record vengono generati tramite l'utilizzo coordinato di Jira, Google Sheets e Grafana che porta alla creazione di un cruscotto di qualità. Si rimanda a @tecnologie_controllo per ulteriori dettagli.
 
-  + mantenere, archiviare e distribuire i rapporti;
+  + *mantenere, archiviare e distribuire i rapporti*:
+    - in sede di retrospettiva il Verificatore è tenuto ad aggiornare il gruppo riguardo le attività di Controllo della Qualità dello sprint in corso, servendosi dell'aiuto del cruscotto di qualità.
```

Comment:
Da questo punto sembra che il cruscotto di qualità sia quasi prerogativa del Verificatore, quando in realtà è uno strumento sempre disponibile a tutti i membri del gruppo proprio perchè *deve* informare *tutti* dello stato di avanzamento dello sprint/del progetto. Quindi per quanto riguarda l'aspetto della _distribuzione delle informazioni_ specificherei bene questo.

Reply:
modifiche fatte